validate-differential-expression/README.md

```diff
@@ -0,0 +1,79 @@
+# refine.bio Examples: Validate Differential Expression
+
+These notebooks compare the differential expression results from a refine.bio dataset to an non-refine.bio dataset that both have a similar experimental set up.
+
+You may obtain differential expression results from your own data or a collaborator’s data.
```

Comment:
I would move from "you may obtain" to "if you have ... it is good practice to..."